src/EnergyPlus/WeatherManager.cc

```diff
@@ -3925,7 +3922,7 @@ namespace Weather {
         state.dataWeather->wvarsHrTsTomorrow.allocate(state.dataGlobal->NumOfTimeStepInHour, Constant::HoursInDay);
     }
 
-    void CalculateDailySolarCoeffs(EnergyPlusData &state,
+    void CalculateDailySolarCoeffs(EnergyPlusData const &state,
```

Comment:
[src/EnergyPlus/WeatherManager.cc:3928]:(style),[constParameterReference],Parameter 'state' can be declared as reference to const